s/features/notes/components/notes/xiome-notes.ts

```diff
@@ -1,156 +1,176 @@
-
-import styles from "./xiome-notes.css.js"
-import {makeNotesModel} from "../../models/notes-model.js"
-import {renderOp} from "../../../../framework/op-rendering/render-op.js"
-import {ComponentWithShare, mixinStyles, html} from "../../../../framework/component.js"
-import {ops} from "../../../../framework/ops.js"
+import styles from "./xiome-notes.css.js";
```

Comment:
the auto-formatting you used has changed this code so much, that we can't tell what substiantial changes have been made, making this very difficult to review

Reply:
I'll revert the formatting